pyvista/plotting/plotting.py

```diff
@@ -3794,7 +3794,7 @@ def add_silhouette(self, mesh, params=None):
         if not isinstance(mesh, pyvista.PolyData):
             raise TypeError(f"Expected type is `PolyData` but {type(mesh)} was given.")
 
-        if isinstance(silhouette_params["decimate"], float):
+        if silhouette_params["decimate"]:
```

Comment:
I was going to point out how this skips 0 decimation target and we should test for `None` instead (assuming that's the default) and let `decimate()` do what it wants, but I realised that 0 target is almost a no-op anyway. It will remove data arrays, but that can't be something anyone relies on.

Approving.

Reply:
Right, 0 should be a no-op, and thus the filter shouldn't be used in that scenario. As for data array removal, data arrays aren't using with the silhouette plotting

Reply:
Good point!